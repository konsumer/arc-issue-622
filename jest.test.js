// this tests if arc tables & sandbox will work in jest

/* global describe, it, expect, beforeAll, afterAll */
const arc = require('@architect/functions')
const sandbox = require('@architect/sandbox')

beforeAll(async () => {
  await sandbox.start({ quiet: true })
})
afterAll(async () => {
  await sandbox.end()
})

describe('jest', () => {
  it('be able to get table-list', async () => {
    const data = await arc.tables()
    expect(Object.keys(data)).toMatchSnapshot()
  })
})
