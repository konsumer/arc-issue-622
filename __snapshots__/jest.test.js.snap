// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`jest be able to get table-list 1`] = `
Array [
  "arc-sessions",
  "jesttest",
  "reflect",
  "_name",
]
`;
